spec/factories/topics.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :topic do
+    title "MyString"
+    user nil
```

Comment:
nil is the default, so you can remove this line. You might also change the `title` to something more descriptive than "MyString"